examples/platform/silabs/efr32/rs911x/wifisdk.gni

```diff
@@ -0,0 +1,41 @@
+import("//build_overrides/chip.gni")
```

Comment:
How does this translate to `examples/platform/silabs/efr32/rs911x/rs9117.gni` I am confused as this one sets the same variables.

is rs9117.gni still used?

Reply:
This was added because of manual error. Deleted the  file `wifisdk.gni `and we are going to modify `rs9117.gni` for Siwx917 NCP with wifi-sdk.

Reply:
@bhmanda-silabs Let's keep this PR for 917 SoC porting only